src/main/java/testapp/core/User.java

```diff
@@ -1,30 +1,40 @@
 package testapp.core;
 
-public class User {
-    private int id;
+import com.google.appengine.api.datastore.Entity;
+
+import java.io.Serializable;
+
+public class User implements Serializable {
+    private long ucn;
     private String name;
```

Comment:
I see how that might be confusing, thinking of the abstraction of human and yet you found the connection of one databases id, which would be the so called ucn, how about our databases id, what made you think that the entyties id did not belong to the user?